dubbo-common/src/test/java/org/apache/dubbo/common/extension/wrapper/impl/DogHelloSPI.java

```diff
@@ -0,0 +1,14 @@
+package org.apache.dubbo.common.extension.wrapper.impl;
+
+import org.apache.dubbo.common.extension.Activate;
+import org.apache.dubbo.common.extension.wrapper.HelloSPI;
+
+@Activate(order = 1)
+public class DogHelloSPI implements HelloSPI {
+
+	@Override
+	public void say(String say) {
+		System.out.println("dog say:" + say);

```

Comment:
`System.out.println` is not helpful for unit test, please remove it and use `Assert` to check if work well